src/components/input/styled.js

```diff
@@ -58,7 +58,7 @@ export const LabelRow = styled(Flex).attrs({
 })`
   font-style: normal;
   font-weight: bold;
-  font-size: 14px;
+  font-size: ${({ size }) => (size === "tiny" ? "12px" : "14px")};
```

Comment:
@burbuli8ra how about `small` keyword, since in `typography.js` we use it for `12px`? (not a strong opinion, i can understand this flagName is more about the input, not just the text)

Reply:
Actually @jacekkolasa you have a valid point here, but `size` was not introduced by me... 😄 
I have used the already existing property (used for example [here](https://github.com/netdata/netdata-ui/blob/master/src/components/input/use-input-styles.js)) and have just updated the missing styles to align with what is on the design system.
As already discussed with @christophidesp we need at some point to review in general the inputs/etc. to align everything and cleanup the code as well!
For the time, I am just going to keep this update instead and discuss a potential refactor when we do the general alignment needed for those components! 😄